src/main/java/de/hpi/bpt/argos/api/entity/EntityEndpoint.java

```diff
@@ -0,0 +1,187 @@
+                getIndexFromParameter(true),
+                getIndexToParameter(true));
+    }
+
+    /**
+     * This method returns the basic URI to retrieve child entities.
+     * @param entityId - the id of the entity to be searched for
+     * @param entityTypeId - the id of the entity type to be searched for
+     * @param attributeNames - the names of the attributes to be searched for
+     * @return - the URI to retrieve child entities from
+     */
+    static String getChildEntitiesUri(long entityId, long entityTypeId, List<String> attributeNames) {
+        String attributeNamesAsString = "";
+        for (int i = 0; i < attributeNames.size(); i++) {
+            attributeNamesAsString = attributeNamesAsString + attributeNames.get(i);
+            if (i != attributeNames.size() - 1) {
+                attributeNamesAsString = attributeNamesAsString + "+";
+            }
+        }
+        return getChildEntitiesBaseUri()
+                .replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"))
+                .replaceAll(getEntityTypeIdParameter(true), Objects.toString(entityTypeId, "0"))
+                .replaceAll(getAttributeNamesParameter(true), attributeNamesAsString);
+    }
+
+    /**
+     * This method returns the basic URI to retrieve an entity.
+     * @param entityId - the id of the entity to be searched for
+     * @return - the URI to retrieve the entity from
+     */
+    static String getEntityUri(long entityId) {
+        return  getEntityBaseUri().replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"));
+    }
+
+    /**
+     * This method returns the basic URI to retrieve the event types of an entity.
+     * @param entityId - the id of the entity to be searched for
+     * @return - the URI to retrieve event types from
+     */
+    static String getEventTypesOfEntityUri(long entityId) {
+        return  getEventTypesOfEntityBaseUri().replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"));
+    }
+
+    /**
+     * This method returns the basic URI to retrieve the events of an entity.
+     * @param entityId - the id of the entity to be searched for
+     * @param entityTypeId - the id of the entity type to be searched for
+     * @param fromIndex - the index of the start of the entities to be searched for
+     * @param toIndex - the index of the end of the entities to be searched for
+     * @return - the URI to retrieve events from
+     */
+    static String getEventsOfEntityUri(long entityId, long entityTypeId, int fromIndex, int toIndex) {
+        return getEventsOfEntityBaseUri()
+                .replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"))
+                .replaceAll(getEntityTypeIdParameter(true), Objects.toString(entityTypeId, "0"))
+                .replaceAll(getIndexFromParameter(true), Objects.toString(fromIndex, "0") )
+                .replaceAll(getIndexToParameter(true), Objects.toString(toIndex, "0") );

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') ')' is preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AParen_Pad_BP2016)